### ISSUE TITLE
dataset spilt

### ISSUE THREAD

**Issue by tangwenlongNO1** [immuno121/ALS4GAN#5]
Can you give me the training set and validation set, or the code to split the dataset

**Comment by tangwenlongNO1** [comment]
for ucm

**Comment by tangwenlongNO1** [comment]
twl102841@gmail.com

**Comment by MajidHeydari** [comment]
Codes are not executed. Is it possible to check the code?
